tests/ha/rear_restore.pm

```diff
@@ -23,7 +24,18 @@ sub run {
     assert_screen('rear-boot-screen');
     send_key_until_needlematch('rear-recover-selected', 'up');
     send_key 'ret';
-    $self->wait_boot_past_bootloader;
+    # Handle the output error for tty0 in ppc64, more details here TEAM-9655
+    if (is_ppc64le) {
+        wait_screen_change { sleep 10 };
```

Comment:
Is this line necessary here? 
If I understand correctly it just waits 10s after pressing enter in bootloader and screen changes. 
Wouldn't `check_screen` timeout do the job itself? 
I am just thinking how to remove arbitrary sleep call.

Reply:
From https://open.qa/api/testapi/#_wait_screen_change, the code to `wait_screen_change` should be used to cause a change in the screen. `sleep 10` will never do that.

I tend to agree with @lpalovsky that this line is not necessary and the `check_screen` that follows would be enough.

Reply:
Ah ok.. I misunderstood that one. 
Still not necessary, but this one could work right?  `wait_screen_change(send_key 'ret';)`